### ISSUE TITLE
Feature Request: OnPress to Context Menu

### ISSUE THREAD

**Issue by ChristopherGabba** [mpiannucci/react-native-context-menu-view#114]
Right now, if you want the context menu to appear, you have to use a `longPress` to get it to pop up. This ultimately results in you needing to add a "Pressable" or "TouchableOpacity" inside the context menu to get a normal "onPress" function for whatever the context is supporting. 

Would be great to have just a normal "onPress" to capture normal press events.

Thanks,
Chris

**Comment by mpiannucci** [comment]
You can show the menu by using it in menu mode, but the preview won't show.

iOS as a platform doesn't support anything other than long press to trigger context previews

**Comment by ChristopherGabba** [comment]
@mpiannucci I am not talking about triggering the context view. I'm talking about just adding a basic `onPress` function for other utilities. Right now, let's say you have an image that you want to add a context menu to. You wrap it in the context menu and now longPress triggers the context menu.

What if you want another function to navigate to a FullImageScreen to show the image just if it's tapped quickly. 

Right now you have to wrap the image with `Touchable` then wrap that with a `ContextMenu` just so you can get an `onPress` function. Would be great to not need the `Touchable` at all and just have a basic `onPress` function.

**Comment by mpiannucci** [comment]
Yes this API would be great, but I haven't found a way to trigger the context menu view programmatically and the iOS system handles the long press behavior automatically. I'm open to PRs tho. 

BTW, if you are just looking to show the menu on press (not on long press) then just set `dropdownMenuMode={true}` on `ContextMenu`

**Comment by ChristopherGabba** [comment]
@mpiannucci Thanks, I didn't see that property, although it is still not quite what I'm looking for. Throw aside opening the context menu, I'm just looking for an extra `onPress` function for all the children that I could use for other things.

**Comment by indapublic** [comment]
+1, I have TouchableOpacity under ContextMenu and it will lead `onPress` fire when I want to open context menu
